functions/toCamelCase/toCamelCase.js

```diff
@@ -0,0 +1,13 @@
+export const toCamelCase = (str) => {
+  if (typeof str !== 'string') {
+    return '';
+  }
```

Comment:
If you have one instruction, you can shorten the code:

if (typeof str !== 'string') return '';